src/modules/site-deal/site-deal.controller.create-facility-folder.test.ts

```diff
@@ -2,30 +2,30 @@ import { CreateFacilityFolderGenerator } from '@ukef-test/support/generator/crea
 import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
 import { when } from 'jest-when';
 
-import { DealFolderService } from './deal-folder.service';
+import { DealFolderCreationService } from './deal-folder-creation.service';
+import { FacilityFolderCreationService } from './facility-folder-creation.service';
 import { SiteDealController } from './site-deal.controller';
-import { SiteDealService } from './site-deal.service';
 
 describe('SiteDealController', () => {
   const valueGenerator = new RandomValueGenerator();
-  const siteDealService = new SiteDealService(null, null, null, null);
+  const siteDealService = new FacilityFolderCreationService(null, null, null, null);
```

Comment:
Should this variable also be renamed (to `facilityFolderCreationService`)?

Reply:
Good spot

Reply:
Changed!